src/HidLibrary/unit/Tuple_VidPid.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace HidLibrary.unit
```

Comment:
I've been away from active open-source C# development for a long time, so I may just be behind on naming conventions. What was the purpose for the "unit" namespace and why did you choose that name for this?